charts/message-parser/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: message-parser
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
I think our PHDI version is only on `1.0.9` - should we change `appVersion` accordingly?

Reply:
changed